tests/www/signup/test_job_seeker.py

```diff
@@ -78,152 +81,64 @@ def test_job_seeker_signup_situation(self, client):
-        assert nir == job_seeker.jobseeker_profile.nir
-        assert job_seeker.title == "M"
-        assert job_seeker.has_jobseeker_profile
-
-    def test_job_seeker_temporary_nir(self, client):
-        """
-        For the moment, we don't handle temporary social numbers.
-        Skipping NIR verification is allowed if a temporary one should be used instead.
-        """
-
-        # Temporary numbers don't have a consistent format.
-        nir = "1234567895GHTUI"
-
-        url = reverse("signup:job_seeker_nir")
-        post_data = {"nir": nir}
-        response = client.post(url, post_data)
-        assert response.status_code == 200
-        assert not response.context.get("form").is_valid()
-
-        post_data = {"nir": nir, "skip": 1}
-        response = client.post(url, post_data)
-        assertRedirects(response, reverse("signup:job_seeker"))
-        assert global_constants.ITOU_SESSION_NIR_KEY not in list(client.session.keys())
-        assert not client.session.get(global_constants.ITOU_SESSION_NIR_KEY)
+        assert global_constants.ITOU_SESSION_JOB_SEEKER_SIGNUP_KEY in list(client.session.keys())
+        assert client.session.get(global_constants.ITOU_SESSION_JOB_SEEKER_SIGNUP_KEY)
+        assertRedirects(response, reverse("signup:job_seeker_credentials"))
 
-        # Temporary NIR is not stored with user information.
-        url = reverse("signup:job_seeker")
+        url = reverse("signup:job_seeker_credentials")
         response = client.get(url)
         assert response.status_code == 200
-        # Since no NIR was provided (or it was a temporary number), suggest nothing
-        assert response.context["form"]["title"].initial is None
-
-        address_line_1 = "Test adresse"
-        address_line_2 = "Test adresse complémentaire"
-        post_code = self.city.post_codes[0]
 
         post_data = {
-            "title": "M",
-            "first_name": "John",
-            "last_name": "Doe",
-            "email": "john.doe+2@company.com",
             "password1": DEFAULT_PASSWORD,
             "password2": DEFAULT_PASSWORD,
-            "address_line_1": address_line_1,
-            "address_line_2": address_line_2,
-            "post_code": post_code,
-            "city_name": self.city.name,
-            "city": self.city.slug,
         }
-
         response = client.post(url, data=post_data)
-        assert response.status_code == 302
         assertRedirects(response, reverse("account_email_verification_sent"))
 
-        job_seeker = User.objects.get(email=post_data["email"])
-        assert job_seeker.jobseeker_profile.nir == ""
-        assert job_seeker.title == "M"
-        assert job_seeker.has_jobseeker_profile
+        # Test session cleanup
+        assert global_constants.ITOU_SESSION_JOB_SEEKER_SIGNUP_KEY not in list(client.session.keys())
+        assert not client.session.get(global_constants.ITOU_SESSION_JOB_SEEKER_SIGNUP_KEY)
```

Comment:
la seconde ligne ne sert à rien vu qu'on vérifie déjà que la clé n'est pas dans la session, mais tu pourrais aussi remplacer les 2 lignes par  
`assert client.session.get(global_constants.ITOU_SESSION_JOB_SEEKER_SIGNUP_KEY) is None`